control-admin/src/main/java/fi/nls/oskari/control/admin/SearchWFSChannelActionHandler.java

```diff
@@ -71,7 +72,7 @@ public void handleGet(ActionParameters params)
                 }
             }
         } catch (Exception ex){
-            log.error(org.apache.commons.lang.exception.ExceptionUtils.getStackTrace(ex));
+            log.error(ExceptionUtils.getStackTrace(ex));
             throw new ActionParamsException("Couldn't get WFS search channels");
```

Comment:
Just add the cause exception instead? I know only import changed here, but I don't think we need the ExceptionUtils here?
```suggestion
            throw new ActionParamsException("Couldn't get WFS search channels", ex);
```